pages/api/auth/[...nextauth].ts

```diff
@@ -68,3 +84,94 @@ export default async function auth(req: NextApiRequest, res: NextApiResponse) {
     secret: process.env.SECRET,
   })
 }
+
+function html({ url, host, email }) {
+  const escapedEmail = `${email.replace(/\./g, '&#8203;.')}`
+  const escapedHost = `${host.replace(/\./g, '&#8203;.')}`
+  return `
+  <html>
```

Comment:
can u move this out to a constants file or sum. i would like to keep nextauth file as clean as poss

Reply:
done, moved these functions to a different file to be more clean

Reply:
NAB, but u think u can make it a ts file? 😂😂😂 sry